internal/gotest/result.go

```diff
@@ -111,11 +111,33 @@ func (a *resultAggregator) Accept(e event) error {
 	return nil
 }
 
+// filterBuildWarnings returns a copy of the supplied events map after removing
+// any resultKeys that only contain "build-output" actions. These actions
+// without a cooresponding build-fail event are just build warnings and do not
```

Comment:
```suggestion
// without a corresponding build-fail event are just build warnings and do not
```
A linter is unhappy with the spelling here:
```
go run github.com/golangci/golangci-lint/cmd/golangci-lint@v1.64.8 run
internal/gotest/result.go:116:14: `cooresponding` is a misspelling of `corresponding` (misspell)
// without a cooresponding build-fail event are just build warnings and do not
             ^
exit status 1
```